whisper/whisperv5/filter_test.go

```diff
@@ -132,6 +132,38 @@ func TestInstallFilters(t *testing.T) {
 	}
 }
 
+func TestInstallSymKeyGeneratesHash(t *testing.T) {
```

Comment:
Do we need a test that checks if a field is set? I would expect that there is a test with symmetric communication that fails if there is something not properly initialized?

Reply:
I see two questions here. The first one is, "do we need a UNIT test that check if a field is properly set?", I would answer 'yes' because unit tests should have an extensive coverage. The second question is: "should not there be an END TO END test that fails if that field isn't properly set", the answer is 'yes' again, and that is indeed missing. I will add such a test, but this one should remain.